src/parser/index.ts

```diff
@@ -43,6 +43,7 @@ import ClassicMageArcane from 'analysis/classic/mage/arcane';
 import ClassicPaladinHoly from 'analysis/classic/paladin/holy';
 import ClassicPriest from 'analysis/classic/priest';
 import ClassicShaman from 'analysis/classic/shaman';
+import ClassicHunter from 'analysis/classic/hunter';
```

Comment:
Should be `ClassicHunterSurvival` and point to `analysis/classic/hunter/survival` after file re-arrange.

Reply:
done